docs/tails_printing_guide.rst

```diff
@@ -7,12 +7,43 @@ potentially untrusted network. This is why **we strongly recommend using
 a printer that does not have WiFi or Bluetooth**, and hooking up to it
 using a regular USB cable to print.
 
-Normally, any printer should work with Tails "out of the box." Most
-difficulties stem from not selecting the right driver (extra software
-needed for the printer and computer to communicate). Luckily, Tails has
-a large number of drivers for just about any popularly manufactured
-printer on hand, without even having to download new drivers from the
-web.
+Installing and Printing via the Tails GUI
+-----------------------------------------
+
+Normally, any printer should work with Tails "out of the box." Let's look at
```

Comment:
Agreed. Your suggestion was great, so I integrated that, along with a description of the Linux Foundation's OpenPrinting database, and how to best use that for identifying other potentially suitable printers. (H/T @fowlslegs)